api/src/main/java/org/pmiops/workbench/BlockscoreService.java

```diff
@@ -1,11 +1,18 @@
 package org.pmiops.workbench;
 
+import com.blockscore.models.Address;
 import com.blockscore.models.PaginatedResult;
 import com.blockscore.models.Person;
+import org.pmiops.workbench.model.BlockscoreVerificationStatus;
 
 /**
  * Encapsulate API for interfacing with Blockscore identity verification service.
  */
 public interface BlockscoreService {
   public PaginatedResult<Person> listPeople();
+  public Person createPerson(
+    String firstName, String lastName, Address address, String dobString,
+    String documentType, String documentNumber);
+  public Person retrievePerson(String blockscoreId);
```

Comment:
This isn't used anywhere. Remove for now?

Reply:
Thank you. Junk from previous iteration.